libcrux-ml-kem/src/polynomial.rs

```diff
@@ -14,8 +14,8 @@ pub(crate) const ZETAS_TIMES_MONTGOMERY_R: [i16; 128] = [
 pub(crate) const VECTORS_IN_RING_ELEMENT: usize =
     super::constants::COEFFICIENTS_IN_RING_ELEMENT / FIELD_ELEMENTS_IN_VECTOR;
 
-#[cfg_attr(eurydice, derive(Clone, Copy))]
-#[cfg_attr(not(eurydice), derive(Clone))]
+// XXX: We don't want to copy this. But for eurydice we have to have this.
```

Comment:
Right, that's a little confusing. Before, we needed copy only for eurydice because of the `cloop` macro, which rewrites iterators into plain `for` loops for eurydice. Now we have issues with `from_fn` in eurydice, hence use `[Default; L]` style array initialisation, which requires `Copy`. But that's also only because of eurydice 😔